### ISSUE TITLE
phpstorm 8.0.2 EAP 139.584

### ISSUE THREAD

**Issue by infabo** [enriquepiatti/Magicento#143]
2014-11-28 09:35:58,020 [  83802]   INFO - ellij.project.impl.ProjectImpl - Error saving project 
java.lang.NoClassDefFoundError: com/intellij/ide/reporter/ConnectionException

```
at com.magicento.MagicentoSettings.getState(MagicentoSettings.java:137)

at com.magicento.MagicentoSettings.getState(MagicentoSettings.java:17)

at com.intellij.openapi.components.impl.stores.ComponentStoreImpl.a(ComponentStoreImpl.java:125)

at com.intellij.openapi.components.impl.stores.ComponentStoreImpl.startSave(ComponentStoreImpl.java:108)

at com.intellij.openapi.components.impl.stores.StoreUtil.doSave(StoreUtil.java:40)

at com.intellij.openapi.project.impl.ProjectImpl.save(ProjectImpl.java:357)

at com.intellij.ide.SaveAndSyncHandlerImpl.saveProjectsAndDocuments(SaveAndSyncHandlerImpl.java:149)

at com.intellij.ide.SaveAndSyncHandlerImpl$3.onFrameDeactivated(SaveAndSyncHandlerImpl.java:103)

at com.intellij.ide.FrameStateManagerImpl.a(FrameStateManagerImpl.java:87)

at com.intellij.ide.FrameStateManagerImpl.access$500(FrameStateManagerImpl.java:32)

at com.intellij.ide.FrameStateManagerImpl$2$1.run(FrameStateManagerImpl.java:72)

at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)

at com.intellij.util.Alarm$Request$1.run(Alarm.java:351)

at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:318)

at java.awt.event.InvocationEvent.dispatch(Unknown Source)

at java.awt.EventQueue.dispatchEventImpl(Unknown Source)

at java.awt.EventQueue.access$400(Unknown Source)

at java.awt.EventQueue$3.run(Unknown Source)

at java.awt.EventQueue$3.run(Unknown Source)

at java.security.AccessController.doPrivileged(Native Method)

at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)

at java.awt.EventQueue.dispatchEvent(Unknown Source)

at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:748)

at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:577)

at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:384)

at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)

at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)

at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)

at java.awt.EventDispatchThread.pumpEvents(Unknown Source)

at java.awt.EventDispatchThread.pumpEvents(Unknown Source)

at java.awt.EventDispatchThread.run(Unknown Source)
```

Caused by: java.lang.ClassNotFoundException: com.intellij.ide.reporter.ConnectionException PluginClassLoader[Magicento, 2.8.6]

```
at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:68)

at java.lang.ClassLoader.loadClass(Unknown Source)

... 31 more
```

**Comment by enriquepiatti** [comment]
https://github.com/enriquepiatti/Magicento/issues/140

**Comment by enriquepiatti** [comment]
solved in Magicento v2.9.0
